htdocs/commande/list.php

```diff
@@ -573,6 +576,14 @@
 		$moreforfilter .= $formother->select_categories('customer', $search_categ_cus, 'search_categ_cus', 1);
 	 	$moreforfilter .= '</div>';
 	}
+	if (!empty($conf->expedition->enabled)){
```

Comment:
By default there is no warehouse on order so we should not enter this part of code by default.
May be you forget a test on the constant that enable the feature to manage warehouse on orders ?

Reply:
I add constant for enable code